listener.go

```diff
@@ -60,17 +62,18 @@ func processSnapshot(snapshot *common.Snapshot) {
 		log.Panicf("Error starting transaction: %v", err)
 	}
 
+	// ensure that no new database updates are made on old database
```

Comment:
dbMux must be used in all plugins that take snapshots, yes. That's part of the pattern documented here: https://github.com/30x/apidApigeeSync/blob/master/README.md#apigeesync-dependent-plugins